app/common/data/interfaces/temporary.py

```diff
@@ -0,0 +1,30 @@
+"""
+This module contains temporary functions that are used to support the scaffolding of the core platform functionality.
+
+We anticipate that they should not be used by any of the real domains themselves (eg apply, assess, monitor), but are
+required to support the technical build-out of the platform.
+
+This file should be removed once the scaffolding is complete and some domain skins are in place.
+
+The only place that should import from here is the `app.developers` package.
+"""
+
+from pydantic import UUID4
```

Comment:
Are we intentionally using pydantics UUID4 vs. the uuid lib? Sorry if I've missed something and thats obvious

Reply:
Hm good query. I've seen this done elsewhere and didn't query it 🤦‍♂️  I don't think using this in type hints actually gives us anything over `uuid.UUID` outside of DB models/pydantic classes. Might be one to sweep up later.